### ISSUE TITLE
Review accessibility section of webpage

### ISSUE THREAD

**Issue by pgmccann** [StAResComp/2019-09-23-standrews#3]
We've previously used the default accessibility text in index.md - this should be reviewed.

**Comment by pgmccann** [comment]
Have dicussed with CAPOD and amended accordingly - see f01bd88b7ae0b2f34642a991585e01eafbddefa3
